### DISCUSSION TITLE
Multiple Servers issue?

### DISCUSSION THREAD

**Discussion by Osymoko** [GerryDazoo/Slinger#62] [General]
Hi Gerry , 

You might remember that I started the reddit  stream towards you - I hope you've been able to get some time to yourself since!  There's a lot of very happy and grateful people.

If I may ask another question, I have set up the new code perfectly on my Win 10 desktop and it registers the slingbox all ok  and I can stream internally and externally wonderfully well.  But I have two other laptops set up exactly the same as the files are co-ordintated across them.  But they fail to register (switching on one at a time) even if I switch all firewalls off - is it the case that you can only install one server per slingbox on only one machine? I'm sure that's not the case looking at other answers.

I should add that all three work internally , it is the external route in for the laptops that fail.

 Logs attached :
![InkedCaptureOK](https://user-images.githubusercontent.com/108595850/189607321-11bf0057-5bca-46b7-ac49-eff07c637f42.jpg)

![InkedCaptureNOTOK](https://user-images.githubusercontent.com/108595850/189607816-181c2e21-d648-412c-a64f-8b2aa68a928b.jpg)

**Comment by GerryDazoo** [comment]
When it registers the slingboxes for Server it verifies that the server is reachable from the WAN. Your port maps on your router are pointing to the wrong server, so it can't respond to the "ping" request because they are never received. 
Do you really need to register the boxes? That code is only for the instance where you don't have a static IP and you need to access your server from the WAN. For now you can comment out the Finderids on the servers that don't have proper port maps or you can change the [SERVER] port to something different for each server and then add the appropriate port forwarding rules to your router.

**Reply by Osymoko** [reply]
Thank you, yet again!  As an early adopter I'd forgotten to open the extra port - it all works like a dream and can't wait to test the laptop server on the road!

**Reply by Osymoko** [reply]
Hi Gerry,  bowled over how helpful you have been for everybody .  I feel I am almost at the last point . I have the server running remotely on a surface pro 8 - it is clearly ok.  But I can't find the right address to log into to VLC /MX which i thought would be http://86.xxx.xxx.81:65434/slingbox.  Are you  able to spot what I am doing wrong with the below please?
Attachments removed

**Reply by GerryDazoo** [reply]
Yes, you're using the wrong port number. The server is listening on port 8082 not 65434

**Comment by Osymoko** [comment]
Strange, 8082 is not working either.... I have been trying that as well, and just tried again. Anything else it might possibly be?

**Reply by GerryDazoo** [reply]
Yes, your firewall is blocking connections. If your running VLC on the same box as the server try 127.0.0.1 for the ip address that will bypass the firewall.

**Reply by Osymoko** [reply]
127.0.0.1 on the same remote machine as the server pings ok as expected , but when used in VLC does not work "Your input can't be opened:
VLC is unable to open the MRL 'http://127.0.0.1:8082/slingbox'. Check the log for details."  

I've switched all firewalls off and no luck..

The VLC log tells me nothing..

VLC is unable to open the MRL 'http://127.0.0.1:65434/slingbox'. Check the log for details.
'. Check the log for details.

**Comment by GerryDazoo** [comment]
Two things
1) Remove the finderid from your config.ini you don't need it for local access
2) Please supply the program output so we can see what's going on. Preferable a copy and paste of the actual output and not a screen shot I have trouble making out the details.

**Reply by Osymoko** [reply]
Thanks Gerry. I'll take a couple of days and put a concise note together. It all works internally and with external access to my local server. I can always switch that on remotely anyway so am always covered. Just wanted to crack this externally run server firewall for the full set.

**Reply by Osymoko** [reply]
just if anyone else reads and has the same remote location problem I do, I solved the issue of 127.0.0.1/8082 not working by using the following CMD as admin line, now I am able to both set up the server and watch remotely on the same laptop away from home

netsh http add iplisten 127.0.0.1

just need to solve the firewall problem so it works on other things than the laptop , but just can't see whats omitted.  Ill keep trying,
